### ISSUE TITLE
Google provider not giving the email parameter

### ISSUE THREAD

**Issue by frangeris** [hybridauth/hybridauth#411]
I'm currently using `dev-3.0.0-Remake` because it is the only one that works for me stably, I am working on a signup trying to get user data with google, the process of authentication is performed successfully and I am returned to my callback url correctly but when I try to retrieve the user's email comes trouble, the email is not in `Hybridauth\User\Profile` object but is returned by google api, here is some more detail:

These are the scope I'm using:

```php
    'hybridauth' => [
        'providers' => [
            'google' => [
                'callback' => 'my_callback_url',
                'keys' => ['id' => getenv('GOOGLE_ID'), 'secret' => getenv('GOOGLE_SECRET')],
                'scope' => 'https://www.googleapis.com/auth/plus.login https://www.googleapis.com/auth/plus.me https://www.googleapis.com/auth/userinfo.email https://www.googleapis.com/auth/userinfo.profile'
            ],
            ...
```

I went directly to the [provider of Google](https://github.com/hybridauth/hybridauth/blob/3.0.0-Remake/src/Provider/Google.php) and modify the method [fetchUserEmail](https://github.com/hybridauth/hybridauth/blob/3.0.0-Remake/src/Provider/Google.php#L90) to see if there was the value `email`, something like this:

![method](https://cloud.githubusercontent.com/assets/811381/6543510/c4aac4d6-c4f4-11e4-921c-3ae9ee10154b.PNG)

And this is the result that I get:

![data](https://cloud.githubusercontent.com/assets/811381/6543527/3d1a90cc-c4f5-11e4-9d30-467f5b1a12c0.PNG)

So, the [`Hybridauth\User\Profile`](https://github.com/hybridauth/hybridauth/blob/3.0.0-Remake/src/User/Profile.php) returned by [fetchUserEmail](https://github.com/hybridauth/hybridauth/blob/3.0.0-Remake/src/Provider/Google.php#L90) do not have the email parameter inside `$data` var..

Any help?

**Comment by frangeris** [comment]
I was forced to change the structure of the method  [fetchUserEmail](https://github.com/hybridauth/hybridauth/blob/3.0.0-Remake/src/Provider/Google.php#L90) manually  from this:

```php
	/**
	*
	*/
	protected function fetchUserEmail( $userProfile, $data )
	{
		foreach( $data->filter( 'emails' )->all() as $email )
		{
			if( 'account' == $email->get( 'type' ) )
			{
				$userProfile->email = $email->get( 'value' );
				break;
			}
		}
		return $userProfile;
 	}
```

to this:
```php
	/**
	*
	*/
	protected function fetchUserEmail( $userProfile, $data )
	{
		foreach( $data->get( 'emails' ) as $email )
		{
			if( 'account' == $email->type )
			{
				$userProfile->email = $email->value;

				break;
			}
		}
		
		return $userProfile;
 	}
```

Only in that way I could get the email...

**Comment by AdwinTrave** [comment]
Closed via #415
